gridded_etl_tools/utils/attributes.py

```diff
@@ -284,6 +284,11 @@ def store(self, new_store):
 
         self._store = new_store
 
+    EXTREME_VALUES_BY_UNIT = {"C": (-90, 60), "K": (183.15, 333.15), "F": (-129, 140)}
```

Comment:
We’ve changed C and F to `deg_C` and `deg_F` to be compliant with astropy, need to change this to ensure equality.

Reply:
deg_K?

Reply:
Nope, Kelvin is not a degree, it’s an absolute unit https://sciencenotes.org/why-there-is-no-degree-in-kelvin-temperature/#:~:text=Kelvin%20doesn't%20use%20degrees,you%20include%20a%20degree%20symbol.

Reply:
Well, I've been saying that wrong my entire life.